doc/source/index.rst

```diff
@@ -1,9 +1,33 @@
-Welcome to SUSE Containerized OpenStack
-=======================================
+Welcome to SUSE Containerized OpenStack Technology Preview
+==========================================================
 
-The socok8s project automates SUSE Containerized OpenStack provisioning
-and lifecycle management on SUSE Container as a Service Platform (CaaSP) and
-SUSE Enterprise Storage (SES), using Airship, shell scripts, and Ansible playbooks.
+The socok8s project automates SUSE Containerized OpenStack provisioning and
+lifecycle management on SUSE Container as a Service Platform (CaaSP) and
+SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
+and Ansible playbooks.
+
+As a technology preview, the included features are not supported. They may be
+functionally incomplete, unstable or in other ways not suitable for production
+use. They are mainly included for customer convenience and give customers a
+chance to test new technologies within an enterprise environment.
+
+This release deploys the following set of core OpenStack services:
+
+  * Keystone
+  * Glance
+  * Cinder
+  * Nova
+  * Nuetron
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later, depends on customer
```

Comment:
extraneous comma

Reply:
I debated about it, but it is how it was in the soc7 doc where I stole the language. I agree with you.